RadarSDK/CLLocation+Radar.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CLLocation+Radar.swift
+//  RadarSDK
+//
+//  Copyright © 2022 Radar Labs, Inc. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+public extension CLLocation {
+
+    var isValid: Bool {
```

Comment:
nice! I like this refactor - makes sense as a `CLLocation` extension and comes out cleaner on the callsite